ospd/xml.py

```diff
@@ -111,3 +111,56 @@ def get_elements_from_dict(data: Dict[str, Any]) -> List[Element]:
         responses.append(elem)
 
     return responses
+
+
+class XmlStringHelper:
+    """ Class with methods to help the creation of a xml object in
+    string format.
+    """
+
+    def create_element(self, elem_name: str, end: bool = False) -> bytes:
+        """ Get a name and create the open element of an entity."""
+
+        if end:
+            ret = "</%s>" % elem_name
+        else:
+            ret = "<%s>" % elem_name
+
+        return ret.encode()
```

Comment:
I don't know why bytes are used here. Why not passing strings and converting them only when it is really necessary (i guess when sending them via the socket)?

Reply:
et.tostring() returns always bytes. If we want to change this to be encoded before sending the data via the socket, we have to decode all the data where et.tostring() is used.